tools/Makefile

```diff
@@ -0,0 +1,14 @@
+CC=gcc
```

Comment:
I'm going to move configure to the top level , probably as the last change in this batch.

autotools will then be responsible for:
 - building the dfu tools
 - running the tests
 - generating the documentation.